E01-Geometry/Benas/src/shapes/Rectangle.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+
+class Rectangle extends Shape
+{
+    public float $length;
+    public float $width;
+
+
+    public function __construct(float $length, float $width)
+    {
+        if($length>0) {
+            $this->length = $length;
+        } else{
+            throw new InvalidArgumentException("Length cannot be less than 0");
```

Comment:
Would change these `length` and `width` checks to fail early for better readability.